### DISCUSSION TITLE
How to implement multi-tenant at database/schema level with alembic migration support

### DISCUSSION THREAD

**Discussion by Udayaprasad** [fastapi/fastapi#6212] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Not applicable
```

### Description

Is there any ideal way to implement multi-tenant database support using `fastapi ` and `sqlmodel` frameworks?

### Operating System

Linux, Windows, macOS

### FastAPI Version

0.79.0

### Python Version

3.9

**Comment by iudeen** [comment]
Hi,

This is not a question related to FastAPI, rather SQL or SQLALCHEMY.
Please close this issue and ask in Stackoverflow.

Thanks
